test/addons/ocm-cluster/start

```diff
@@ -122,14 +123,33 @@ def label_cluster(cluster, hub):
 
 
 def wait_for_managed_cluster(cluster, hub):
-    print("Waiting until managed cluster is available")
+    # This takes less then a second, but sometimes it never complete, even if waiting 10 minutes.
+    start = time.monotonic()
+
+    print(f"Waiting until managed cluster '{cluster}' hubAcceptsClient is true")
     kubectl.wait(
         f"managedcluster/{cluster}",
-        "--for=condition=ManagedClusterConditionAvailable",
-        "--timeout=600s",
+        "--for=jsonpath={.spec.hubAcceptsClient}=true",
```

Comment:
This may not be needed since we wait for HubAcceptedManagedCluster - need to ask ocm guys about this.

Reply:
Asked here:
https://kubernetes.slack.com/archives/C01GE7YSUUF/p1712086783211369?thread_ts=1711671907.381919&cid=C01GE7YSUUF

Reply:
Based on ocm code, the extra check can be useful to have, keeping it.